ubtue_Brill.js

```diff
@@ -1,15 +1,15 @@
 {
 	"translatorID": "b2fcf7d9-e023-412e-a2bc-f06d6275da24",
-	"label": "Brill",
+	"label": "ubtue_Brill",
 	"creator": "Madeesh Kannan, Timotheus Kim",
 	"target": "^https?://brill.com/view/journals/",
 	"minVersion": "3.0",
 	"maxVersion": "",
 	"priority": 90,
-	"inRepository": false,
+	"inRepository": true,
 	"translatorType": 4,
 	"browserSupport": "gcsibv",
-	"lastUpdated": "2020-04-01 10:11:25"
+	"lastUpdated": "2021-04-21 09:05:36"
 }
 
 /*
```

Comment:
Shouldn't the '†' handling part as a simple replace oneliner?

Reply:
how should look one line solution?
I have only this one 
```
for (let i in item.creators) {
		item.creators[i].lastName = item.creators[i].lastName.replace('†', '');
		item.creators[i].firstName = item.creators[i].firstName.replace('†', '');
	}
```

Reply:
`firstName` should not be affected.

Reply:
might be happen, if names are not splitted correktly `firstName `